src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -350,12 +352,14 @@ void ACA_OVS_L2_Programmer::execute_ovsdb_command(const std::string cmd_string,
   auto ovsdb_client_end = chrono::steady_clock::now();
 
   auto ovsdb_client_time_total_time =
-          cast_to_nanoseconds(ovsdb_client_end - ovsdb_client_start).count();
+          cast_to_microseconds(ovsdb_client_end - ovsdb_client_start).count();
 
   culminative_time += ovsdb_client_time_total_time;
 
-  ACA_LOG_INFO("Elapsed time for ovsdb client call took: %ld nanoseconds or %ld milliseconds. rc: %d\n",
-               ovsdb_client_time_total_time, ovsdb_client_time_total_time / 1000000, rc);
+  g_total_execute_ovsdb_time += ovsdb_client_time_total_time;
+
+  ACA_LOG_INFO("Elapsed time for ovsdb client call took: %ld microseconds or %ld milliseconds. rc: %d\n",
+               ovsdb_client_time_total_time, ovsdb_client_time_total_time / 1000, rc);
```

Comment:
[Minor] Consider to take the time conversion logic to a small util function so that we don't see /1000, or /1000000 all over the places.

Reply:
fixed.